src/websocket_server/server.ts

```
@@ -0,0 +1,168 @@
+
+        if (closedPlayerId) {
+            removePlayerFromRooms(closedPlayerId);
+            const availableRooms = gameRoomsDB
+                .filter(room => room.roomUsers.length === 1)
+                .map(room => ({
+                    roomId: room.roomId,
+                    roomUsers: room.roomUsers.map(u => ({name: u.name, index: u.index}))
+                }));
+            broadcastToAll(wss, { type: "update_room", data: JSON.stringify(availableRooms), id: 0 });
+            console.log(`[Broadcast] Sent 'update_room' after player ${closedPlayerId} disconnected.`);
+            delete (ws as any).playerId;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)